sdks/python/setup.py

```diff
@@ -232,12 +232,8 @@ def get_portability_package_data():
         # The issue was fixed in version 1.9.2.1
         'crcmod>=1.7,<2.0',
         'orjson<4.0',
```

Comment:
nit: let's move orjson below to make order alphabetical here.

Reply:
Done